tasks/main.yml

```diff
@@ -121,16 +135,19 @@
   copy:
     src: "{{ license_file_path }}"
     dest: /etc/quamotion/.license
+    owner: "{{ appium_user }}"
```

Comment:
Are these permissions needed? I would assume the default permissions are something like read/write for owner + read-only for others?